src/app/_services/auth.service.ts

```diff
@@ -424,7 +424,7 @@ export class AuthService implements CanActivate {
     if (this.getFromStorage('AJIInterceptClosed')) {
       return false
     }
-    return user && user.isLoggedIn && user.preferences.hasOwnProperty('showAJIModalOrBanner') && user.preferences.showAJIModalOrBanner
+    return user && user.isLoggedIn && user.hasOwnProperty('showAJIModalOrBanner') && user.showAJIModalOrBanner == "true"
```

Comment:
```suggestion
    return user && user.isLoggedIn && user.hasOwnProperty('showAJIModalOrBanner') && user.showAJIModalOrBanner
```

Reply:
^ Ignore this suggestion. I think checking against "true" is better in this case.